mla/threeml/data_handlers.py

```diff
@@ -0,0 +1,126 @@
+"""Docstring"""
+
+__author__ = 'John Evans'
+__copyright__ = 'Copyright 2021 John Evans'
+__credits__ = ['John Evans', 'Jason Fan', 'Michael Larson']
+__license__ = 'Apache License 2.0'
+__version__ = '0.0.1'
+__maintainer__ = 'John Evans'
+__email__ = 'john.evans@icecube.wisc.edu'
+__status__ = 'Development'
+
+import dataclasses
+
+import numpy as np
+import numpy.lib.recfunctions as rf
+
+from .. import data_handlers
+from . import spectral
+
+
+@dataclasses.dataclass
+class ThreeMLDataHandler(data_handlers.TimeDependentNuSourcesDataHandler):
+    """Docstring"""
+    injection_spectrum: spectral.BaseSpectrum
+    _injection_spectrum: spectral.BaseSpectrum = dataclasses.field(
+        init=False, repr=False, default=spectral.PowerLaw(1e3, 1e-14, -2))
+    _reduced_reco_sim: np.ndarray = dataclasses.field(init=False, repr=False)
+
+    def __post_init__(self) -> None:
+        """Docstring"""
+        self._reduced_reco_sim = self.cut_reconstructed_sim(
+            self.config['dec_cut_location'], self.config['reco_sampling_width'])

```

Comment:
how is this used differently than the current cut simulation? do you just need it to behave slightly differently?